$NetBSD$

Please see comments in patches/patch-aa before removing this file. 
Thank you!

	-- dotz@irc.pl

--- app/vectors/gimpvectors.c.orig	Mon Mar  1 17:48:54 2004
+++ app/vectors/gimpvectors.c
@@ -545,7 +545,7 @@ gimp_vectors_stroke (GimpItem     *item,
                      GimpObject   *stroke_desc,
                      gboolean      use_default_values)
 {
-  GimpVectors *vectors = GIMP_VECTORS (item);;
+  GimpVectors *vectors = GIMP_VECTORS (item);
   gboolean     retval  = FALSE;
 
   if (! vectors->strokes)
@@ -564,7 +564,7 @@ gimp_vectors_stroke (GimpItem     *item,
   else if (GIMP_IS_PAINT_INFO (stroke_desc))
     {
       GimpImage        *gimage     = gimp_item_get_image (item);
-      GimpPaintInfo    *paint_info = GIMP_PAINT_INFO (stroke_desc);;
+      GimpPaintInfo    *paint_info = GIMP_PAINT_INFO (stroke_desc);
       GimpPaintOptions *paint_options;
       GimpPaintCore    *core;
 
